octue/mixins/taggable.py

```diff
@@ -40,6 +37,19 @@ def __init__(self, tags):
 
         self._tags = self._clean(tags)
 
+    def __str__(self):
+        """ Represents tags as a space delimited string
+        """
+        return self.serialise()
```

Comment:
Is it the docstring or the implementation that needs updating? If I was working with a `Taggable` class, I'd want `.tags` to give me an iterable of tags rather than a string I then have to split.

Reply:
Agreed, I think it's the docstring.

Reply:
Cnonversation online - Taggables should be:
- able to be deserialised from space-delimited tagstring
- be serialised to sorted list of strings so tehat the JSON most closely represents the nature of tags, which is a set.